pages/common/catimg.md

```diff
@@ -0,0 +1,21 @@
+# catimg
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
```

Comment:
```suggestion
> See also: `pixterm`, `chafa`.
```

Reply:
Thanks, I'll update this on my other PRs as well.